nixos/modules/services/network-filesystems/ipfs.nix

```diff
@@ -142,6 +142,35 @@ in
         description = "If set to true, the repo won't be initialized with help files";
       };
 
+      swarmAddrFitlers = mkOption {
+        type = types.listOf types.str;
+        default = [
+          "10.0.0.0/8"
+          "100.64.0.0/10"
+          "169.254.0.0/16"
+          "172.16.0.0/12"
+          "192.0.0.0/24"
+          "192.0.2.0/24"
+          "192.168.0.0/16"
+          "198.18.0.0/15"
+          "198.51.100.0/24"
+          "203.0.113.0/24"
+          "240.0.0.0/4"
+          "100::/64"
+          "2001:2::/48"
+          "2001:db8::/32"
+          "fc00::/7"
+          "fe80::/10"
+        ];
```

Comment:
I think this is the wrong format. You should use the list from https://github.com/ipfs/go-ipfs-config/blob/a412831517bb5ce3672af95e0b63c3633218e80e/profile.go#L24-L44. This is also the list that IPFS itself puts in the config file when executing `ipfs config profile apply server`.